src/main/java/dev/j3fftw/litexpansion/utils/Utils.java

```diff
@@ -51,4 +51,8 @@ public static double perTickToPerSecond(double power) {
             return (1 / ((double) Constants.CUSTOM_TICKER_DELAY / Constants.SERVER_TICK_RATE) * power);
         }
     }
+
+    public static void lxPrefixMessage(Player p, ChatColor color, String message) {
+        p.sendMessage(ChatColor.GRAY + "[LiteXpansion] " + color + message);
```

Comment:
I'd probably just `ChatColors.color`, quicker to write the messahes and generally looks nicer.